index.html

```diff
@@ -142,13 +142,13 @@ <h3> Students </h3>
                 <h1>Connect with Fossasia on Social Media</h1>
                 <p> Be sure to follow us on our social media! </p>
                 <p class="large">
-                    <a href="https://www.facebook.com/fossasia/" class="ti-facebook" target="_blank"></a>
-                    <a href="https://twitter.com/fossasia" class="ti-twitter" target="_blank"></a>
-                    <a href="https://www.instagram.com/fossasia/" target="_blank" class="ti-instagram lgi"></a>
-					<a href="https://www.youtube.com/channel/UCQprMsG-raCIMlBudm20iLQ" target="_blank" class="ti-youtube lgi"></a>
-					<a href="https://github.com/fossasia" target="_blank" class="ti-github lgi"></a>
-					<a href="https://fossasia.org/" target="_blank" class="ti-world lgi"></a>
-					<a href="https://codein.withgoogle.com/organizations/fossasia/" target="_blank" class="ti-settings lgi"></a>
+                    	<a href="https://www.facebook.com/fossasia/" class="ti-facebook" target="_blank"></a>
```

Comment:
@fragm3 is this ok or should the indent look different?

Reply:
One tab please, you have added more than that

Reply:
One tab counting from the `<p class="large">` tag or from the border of the page? Because, at least from my editor, the indent is currently one tab counting from the level of the `<p class="large">`  tag.